src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -149,6 +149,12 @@
   <data name="SettingsValidateErrorTitle" xml:space="preserve">
     <value>Encountered errors while loading user settings</value>
   </data>
+  <data name="KeyboardServiceWarningTitle" xml:space="preserve">
+    <value>Warning:</value>
+  </data>
+  <data name="KeyboardServiceWarningText" xml:space="preserve">
+    <value>The "Touch Keyboard and Handwriting Panel Service" isn't running on your machine. This can prevent the Terminal from receiving keyboard input.</value>

```

Comment:
@cinnamon-msft, could we maybe add a heading in https://docs.microsoft.com/en-us/windows/terminal/troubleshooting about this and link to there from here to show pictures of how to turn it back on?

Reply:
Oh good, i already had to write this up for another thread:
https://github.com/microsoft/terminal/issues/8130#issuecomment-720575268

So I'm just gonna take that and turn that into a subsection on the docs